integration/framework/framework.go

```diff
@@ -328,8 +320,11 @@ func (self shellActions) Run(command string, args ...string) (string, string) {
 		cmd = exec.Command(command, args...)
 	} else {
 		// We must SSH to the remote machine and run the command.
-		args = append(common.GetGCComputeArgs("ssh", self.fm.Hostname().GceInstanceName, "--", command), args...)
-		cmd = exec.Command("gcloud", args...)
+		args = append([]string{self.fm.Hostname().GceInstanceName, "--", command}, args...)
```

Comment:
Because I didn't clean up all the of the Gce specific code in the test runner :p .  Do you think that is important enough to do for this PR?